Dockerfile

```diff
@@ -0,0 +1,66 @@
+
+### 1. Get Linux
+FROM ubuntu:18.04
+
+ENV ROBOT 1.5.0
+ARG ROBOT_JAR=https://github.com/ontodev/robot/releases/download/$ROBOT/robot.jar
+ENV ROBOT_JAR ${ROBOT_JAR}
+ENV DOSDPVERSION 0.13.1
+ENV JENA 3.12.0
+ENV BGR 1.4
+ENV KBOT 1.9.1
+
+
+### 2. Get Java and all required system libraries
+RUN apt-get update && apt-get upgrade -y \
+ && apt-get install -y software-properties-common \
+  build-essential git \
+  openjdk-8-jre openjdk-8-jdk
```

Comment:
It's best practice to put each piece into its own line. Or everything in one single line, but not half this half that.